dtfs-central-api/src/helpers/get-previous-report-period.test.ts

```diff
@@ -37,19 +37,49 @@ describe('getPreviousReportPeriod', () => {
     });
   });
 
-  describe('when a bank is found', () => {
+  describe('when a bank is found and the month has 1 digit', () => {
     beforeEach(() => {
       jest.mocked(getBankById).mockImplementation(jest.fn().mockResolvedValueOnce(mockGetBankByIdResponse));
     });
 
     it('should return the result of getPreviousReportPeriodForBankScheduleByMonth', async () => {
+      const reportPeriod = { start: { month: 2, year: 2024 }, end: { month: 4, year: 2024 } };
+      utilisationReport = UtilisationReportEntityMockBuilder.forStatus('PENDING_RECONCILIATION').withReportPeriod(reportPeriod).build();
       const response = await getPreviousReportPeriod(bankId, utilisationReport);
 
-      const { start } = utilisationReport.reportPeriod;
-      const paddedMonth = String(start.month).padStart(2, '0');
-      const monthFormatted = `${start.year}-${paddedMonth}`;
+      const expected = getPreviousReportPeriodForBankScheduleByMonth(mockGetBankByIdResponse.utilisationReportPeriodSchedule, '2024-02');
 
-      const expected = getPreviousReportPeriodForBankScheduleByMonth(mockGetBankByIdResponse.utilisationReportPeriodSchedule, monthFormatted);
+      expect(response).toEqual(expected);
+    });
+  });
+
+  describe('when a bank is found and the month has 2 digits', () => {
```

Comment:
Nitpick but for clarity given there are multiple months defined in the test:
```suggestion
  describe('when a bank is found and the start month of the report period has 2 digits', () => {
```